web_src/js/components/RepoBranchTagSelector.vue

```diff
@@ -190,16 +190,16 @@ const sfc = {
       }
       this.isLoading = true;
       try {
-        // the "data.defaultBranch" is ambiguous, it could be "branch name" or "tag name"
+        // the "data.defaultSelectedBranch" is ambiguous, it could be "branch name" or "tag name"
```

Comment:
Since you have renamed it .... why not give it a better name? Then it isn't ambiguous and such comment could be removed.

Reply:
`defaultSelected`? 🤔

Reply:
`defaultSelectedRefName`